front/src/modules/simulationResult/components/SpaceTimeChart/utils.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
+
+export const getTrainIdsPackage = (tenIndex: number, trainIds: number[], batchSize: number) => {
```

Comment:
`indexInBatch` may be better than `tenIndex`?

Reply:
Other naming nit: s/Package/Batch/?

Reply:
Done :+1: